config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,24 @@
+# id columns are generated as bigint, and we sometimes use constraint-less integer columns
+# that reference another table. To ensure we use bigint for those columns, this check raises
+# if a column is added that uses :integer and the column_name ends with "_id".
+#
+# Some "_id" columns may not be references to other tables and may not need to be bigint.
+# To exclude a table/column from this check add a `[table, column]` item into excluded_columns:
+#
+# Ex: excluded_columns = [[:users, :my_new_id]]
+
+excluded_columns = []
```

Comment:
Let's make this a constant? Possibly namespace it so we don't leak it globally?

```
class StrongMigrations
  EXCLUDED_COUMNS = []
end
```

Reply:
good call